iphone/Classes/TiUIiPadPopoverProxy.m

```diff
@@ -405,6 +359,11 @@ - (void)proxyDidRelayout:(id)sender
   }
 }
 
+- (UIModalPresentationStyle)adaptivePresentationStyleForPresentationController:(UIPresentationController *)controller
+{
+  return UIModalPresentationNone;
```

Comment:
Does this need to be hardcoded?

Reply:
YES, UIModalPresentationNone tells the presentation controller to use the original presentation style which in our case will display a popover.